test/puppetlabs/http/client/sync_plaintext_test.clj

```diff
@@ -236,6 +258,35 @@
             (is (= "Hello, World!" (slurp (:body response))))))
         (.close client)))))
 
+(deftest java-api-cookie-test
+  (testlogging/with-test-logging
+    (testutils/with-app-with-config app
+      [jetty9/jetty9-service test-cookie-service]
+      {:webserver {:port 10000}}
+      (let [request-options (RequestOptions. "http://localhost:10000/cookietest")
+            response-options (RequestOptions. "http://localhost:10000/cookiecheck")
```

Comment:
this is super nitpicky but the name 'response-options' seems like a weird variable name here.  You could probably just inline these into the `get` calls and get rid of the `let` block, and then you will avoid one of the two hard problems in computer science :)